components/brave_vpn/brave_vpn_utils.h

```diff
@@ -9,6 +9,7 @@
 #include <string>
 
 #include "build/build_config.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
same as above, but also there's not need to duplicate the same header in .h and .cc

Reply:
same as above, removed in https://github.com/brave/brave-core/pull/15989/commits/c2b399158c4310c6ed1d1d94890a2a0c179f9721